charts/posit-chronicle/templates/NOTES.txt

```diff
@@ -0,0 +1,20 @@
+{{ include "posit-chronicle.fullname" . }} successfully deployed to namespace {{ .Release.Namespace }}
+
+Please visit https://docs.posit.co/chronicle/getting-started/installation/on-kubernetes.html#agent-sidecar for
+additional information on deploying Chronicle agents to monitor Posit products in your cluster.
+{{ if and (not .Values.config.LocalStorage.Enabled) .Values.persistence.enabled }}
+WARNING: Persistence is enabled, but Chronicle local storage is not configured. This may lead to data loss if the pod
+is restarted or rescheduled.
+{{- end }}
+{{ if and .Values.config.LocalStorage.Enabled (not .Values.persistence.enabled) }}
+WARNING: Local storage is enabled, but persistence is not enabled. This may lead to data loss if the pod is restarted or
+rescheduled.
+{{- end }}
+{{ if and (not .Values.config.LocalStorage.Enabled) (not .Values.config.S3Storage.Enabled) }}
+WARNING: No storage backend is configured. Chronicle will not be able to store any data in a persistent or accessible
+location. Consider redeploying with `.Values.config.LocalStorage` or `.Values.config.S3Storage` set to true.
+{{- end }}
+{{ if and .Values.config.LocalStorage.Enabled and .Values.config.S3Storage.Enabled }}
```

Comment:
```suggestion
{{ if and .Values.config.LocalStorage.Enabled .Values.config.S3Storage.Enabled }}
```
I saw the failing lint and see this fixes it.  I found you can run the lint command locally if you:
```
brew install chart-testing yamllint
just lint
```